sympy/functions/special/polynomials.py

```diff
@@ -799,8 +799,8 @@ def eval(cls, n, x):
         else:
             # n is a given fixed integer, evaluate into polynomial
             if n.is_negative:
-                raise ValueError(
-                    "The index n must be nonnegative integer (got %r)" % n)
+                n = -n - S.One
+                return cls._eval_at_order(n, x)
```

Comment:
Instead of repeating the same return statement, put it simpler:
```
if n.is_negative:
    n = -n - S.One
return cls._eval_at_order(n, x)
 ```
Having fewer return statements is better, the code gets easier to track.